quickstartKotlin/src/main/java/com/twilio/video/quickstart/kotlin/VideoActivity.kt

```diff
@@ -0,0 +1,669 @@
+package com.twilio.video.quickstart.kotlin
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.DialogInterface
+import android.content.pm.PackageManager
+import android.media.AudioAttributes
+import android.media.AudioFocusRequest
+import android.media.AudioManager
+import android.os.Build
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.support.design.widget.Snackbar
+import android.support.v4.app.ActivityCompat
+import android.support.v4.content.ContextCompat
+import android.support.v7.app.AlertDialog
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.widget.EditText
+import android.widget.Toast
+import com.koushikdutta.ion.Ion
+import com.twilio.video.*
+import kotlinx.android.synthetic.main.activity_video.*
+import kotlinx.android.synthetic.main.content_video.*
+import java.util.*
+
+
+class VideoActivity : AppCompatActivity() {
+    private val CAMERA_MIC_PERMISSION_REQUEST_CODE = 1
+    private val TAG = "VideoActivity"
+
+    /*
+     * You must provide a Twilio Access Token to connect to the Video service
+     */
+    private val TWILIO_ACCESS_TOKEN = BuildConfig.TWILIO_ACCESS_TOKEN
+    private val ACCESS_TOKEN_SERVER = BuildConfig.TWILIO_ACCESS_TOKEN_SERVER
+
+    /*
+     * Access token used to connect. This field will be set either from the console generated token
+     * or the request to the token server.
+     */
+    private var accessToken: String? = null
```

Comment:
I would prefer to switch this one to `lateinit`, cause it must be initialized anyway